### ISSUE TITLE
Context menu manage artwork shows up blank

### ISSUE THREAD

**Issue by xyzfre** [jurialmunkey/skin.arctic.horizon.2#500]
### Skin section

Home Widgets

### Current Behavior

This only happens on home screen widgets For some TV shows or movies bringing up the context menu manage artwork shows up blank but if I try the same TV show or movie within TMDb H it does work properly ( Nvidia Shield 2019 Pro K19.4)

### Expected Behavior

When bringing up the context menu on home screen widgets and clicking manage artwork to be able to choose different posters etc..

### Steps To Reproduce

1. Bring up the context menu on a home screen widget 2. Click on TMDb H options 3. Choose manage artwork and you get a blank screen

### Screenshots and Additional Info

![D7B2B1E4-4EAC-4756-9291-71DA0E198A14](https://user-images.githubusercontent.com/49699869/193450679-2e363568-5d06-4a1c-a432-e606bd47841a.jpeg)
Nvidia Shield 2019 Pro 

### Checklist

- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by xyzfre** [comment]
I just tried on the PC and this feature works but not on my Nvidia Shield

**Comment by xyzfre** [comment]
![image](https://user-images.githubusercontent.com/49699869/193454045-bc2f8c34-67d9-47b5-b1eb-66d48d71e6bc.jpeg) PC

**Comment by TechXXX** [comment]
I have a clean install with 19.4 here, it doesn't even give me the TMDB helper options entry in the context menu, my own builds do and on one I get a blank screen and on one it works (both nexus nightlies). I've got a workaround for you which I found when coming across this bug ( https://github.com/jurialmunkey/skin.arctic.horizon.2/issues/498 ). If you move right from wiki it also gives you access to artwork (and an ugly codeline;). 

<img width="1728" alt="Screenshot 2022-10-02 at 09 26 18" src="https://user-images.githubusercontent.com/10571873/193453948-2408ef14-4cdc-469f-aab2-d520139cc9c7.png">

p.s. weird behaviour, if you repeat it, it shows up.
